.github/workflows/create-cherry-pick-pr.yml

```diff
@@ -0,0 +1,18 @@
+name: Create cherry-pick PR
+on:
+  pull_request_target:
+    branches:
+      - 'v*.x'
+      - 'master'
+    types: ['closed']
```

Comment:
@michelengelen Do you think we can make cherry-pick possible *after* the PR was merged, but didn't have the "needs cherry-pick" label?
Take this PR for example: https://github.com/mui/mui-x/pull/15255 It was merged, and then we realized we want to cherry-pick it to v7. Since it's already merged, adding the "needs cherry-pick" label is too late.

Ideally, we would have 2 triggers for cherry-pick:
1. PR is merged and has the  "needs cherry-pick" label
2. Label is added to the PR, the PR now has the "needs cherry-pick" label, and it's already merged.

What do you think?

Reply:
That's a great idea. 👍 
It might prove slightly tricky to use because we would probably need to ensure the proper `vX.x` labels are added before adding the `needs cherry-pick` label. 🤔

Reply:
It's already on my list! But good observation! 💪🏻👍🏼

Reply:
An alternative is to have a manual `workflow_dispatch`, with a required `PR # to cherrypick` and `target`